src/visualisation/CSVWriter.cpp

```diff
@@ -14,16 +14,16 @@ static std::string ending = ".csv";
 const static char delimiter = ',';
 
 void CSVWriter::write_numerical(FieldController *field_controller, const std::string& filename) {
```

Comment:
I know it's my fault, but I didn't see it. The & has to be next to the variable name, not the variable type

Reply:
done in 520939d